man/gbz80.7

```diff
@@ -1129,9 +1170,7 @@ Bytes: 1
 .Pp
 Flags: None affected.
 .Ss OR A,r8
-Store into
-.Sy A
-the bitwise OR of the value in
+Bitwise OR between the value in
```

Comment:
Oh? I'm surprised that you felt it was preferable to not call out explicitly where the result gets stored. Why's that?

Reply:
This mimics the wording for `AND` and `XOR`: "Bitwise *OP* between the *argument* and **A**."

However, you're right, it could be better. The arithmetic operations are worded as "Add the *argument* to **A**", and "Subtract the *argument* from **A**". Maybe the bitwise ones should be "Bitwise *OP* **A** with the *argument*".

Reply:
Hmm, maybe, but IMO that's still a little tenuous and thus leaves the door open to confusion. I'd like the destination register pointed out explicitly.

Reply:
I like the "Store into ..." wording, because it also keeps the order of the arguments.